### PR TITLE
New package: Telegram.Unigram version 10.2.3.9268

### PR DESCRIPTION
**Pull request by rakleed**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141806)

### CODE REVIEW

**Comment by rakleed** [comment]
@stephengillie when I try testing manifest, I get `Installer failed with exit code: 0x80070002 : The system cannot find the file specified.` Is it a problem with winget-cli or Unigram?

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-03-02 15:02:39.357 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(161)\WindowsPackageManager.dll!00007FFA072D3F75: (caller: 00007FFA0722A720) Exception(1) tid(1774) 80073CF3 Package failed updates, dependency or conflict validation.
> 2024-03-02 15:02:39.462 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FFA072D2D2F: (caller: 00007FFA072D432C) Exception(2) tid(1774) 80073CF1 Package was not found.
>     Msg:[Operation failed: Windows cannot remove TelegramFZ-LLC.Unigram_10.2.3.9268_neutral_~_1vfw5zm9jmzqy because the current user does not have that package installed. Use Get-AppxPackage to see the list of packages installed.] 
> 2024-03-02 15:02:39.462 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(126)\WindowsPackageManager.dll!00007FFA07448931: (caller: 00007FFA072D5853) LogHr(1) tid(1774) 80073CF1 Package was not found.
>     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FFA072D2D2F: (caller: 00007FFA072D432C) Exception(2) tid(1774) 80073CF1 Package was not found.
> 2024-03-02 15:02:39.471 [CLI ] MSIX installer failed: 2147958003

(Automated response - build 859.)

File: manifests/t/Telegram/Unigram/10.2.3.9268/Telegram.Unigram.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+PackageVersion: 10.2.3.9268
+Platform:
+- Windows.Universal
+MinimumOSVersion: 10.0.18362.0
+InstallerType: msix
+PackageFamilyName: TelegramFZ-LLC.Unigram_1vfw5zm9jmzqy
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.UI.Xaml.2.8
+Installers:
```
Comment (stephengillie) [inline manifests/t/Telegram/Unigram/10.2.3.9268/Telegram.Unigram.installer.yaml:14]:
```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 860.)

**Comment by rakleed** [comment]
@stephengillie what is wrong now?

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-03-04 09:56:40.688 [CORE] Deployment operation #2: Windows cannot install package TelegramFZ-LLC.Unigram_10.2.3.9268_x64__1vfw5zm9jmzqy because this package depends on a framework that could not be found. Provide the framework "Microsoft.VCLibs.140.00" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 14.0.30704.0, along with this package to install. The frameworks with name "Microsoft.VCLibs.140.00" currently installed are: {Microsoft.VCLibs.140.00_14.0.27323.0_x64__8wekyb3d8bbwe Microsoft.VCLibs.140.00_14.0.27810.0_x64__8wekyb3d8bbwe Microsoft.VCLibs.140.00_14.0.29231.0_x64__8wekyb3d8bbwe Microsoft.VCLibs.140.00_14.0.30035.0_x64__8wekyb3d8bbwe Microsoft.VCLibs.140.00_14.0.30035.0_x86__8wekyb3d8bbwe}
> 2024-03-04 09:56:40.689 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(161)\WindowsPackageManager.dll!00007FFEADE93F75: (caller: 00007FFEADDEA720) Exception(1) tid(594) 80073CF3 Package failed updates, dependency or conflict validation.
> 2024-03-04 09:56:40.808 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FFEADE92D2F: (caller: 00007FFEADE9432C) Exception(2) tid(594) 80073CF1 Package was not found.
> 2024-03-04 09:56:40.808 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(126)\WindowsPackageManager.dll!00007FFEAE008931: (caller: 00007FFEADE95853) LogHr(1) tid(594) 80073CF1 Package was not found.
>     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FFEADE92D2F: (caller: 00007FFEADE9432C) Exception(2) tid(594) 80073CF1 Package was not found.
> 2024-03-04 09:56:40.813 [CLI ] MSIX installer failed: 2147958003

(Automated response - build 861.)

**Comment by rakleed** [comment]
@stephengillie what is wrong again?

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-03-04 10:37:16.537 [CORE] Deployment operation #3: error 0x80073D02: Unable to install because the following apps need to be closed Microsoft.DesktopAppInstaller_1.21.2771.0_x64__8wekyb3d8bbwe Microsoft.XboxGamingOverlay_5.721.5282.0_x64__8wekyb3d8bbwe.
> 2024-03-04 10:37:22.071 [CORE] Deployment operation #4: Windows cannot install package TelegramFZ-LLC.Unigram_10.2.3.9268_x64__1vfw5zm9jmzqy because this package depends on a framework that could not be found. Provide the framework "Microsoft.VCLibs.140.00" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 14.0.30704.0, along with this package to install. The frameworks with name "Microsoft.VCLibs.140.00" currently installed are: {Microsoft.VCLibs.140.00_14.0.27323.0_x64__8wekyb3d8bbwe Microsoft.VCLibs.140.00_14.0.27810.0_x64__8wekyb3d8bbwe Microsoft.VCLibs.140.00_14.0.29231.0_x64__8wekyb3d8bbwe Microsoft.VCLibs.140.00_14.0.30035.0_x64__8wekyb3d8bbwe Microsoft.VCLibs.140.00_14.0.30035.0_x86__8wekyb3d8bbwe}
> 2024-03-04 10:37:22.071 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(161)\WindowsPackageManager.dll!00007FF99F0A3F75: (caller: 00007FF99EFFA720) Exception(1) tid(de8) 80073CF3 Package failed updates, dependency or conflict validation.
> 2024-03-04 10:37:22.214 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FF99F0A2D2F: (caller: 00007FF99F0A432C) Exception(2) tid(de8) 80073CF1 Package was not found.
> 2024-03-04 10:37:22.215 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(126)\WindowsPackageManager.dll!00007FF99F218931: (caller: 00007FF99F0A5853) LogHr(1) tid(de8) 80073CF1 Package was not found.
>     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FF99F0A2D2F: (caller: 00007FF99F0A432C) Exception(2) tid(de8) 80073CF1 Package was not found.
> 2024-03-04 10:37:22.224 [CLI ] MSIX installer failed: 2147958003

(Automated response - build 861.)

**Comment by stephengillie** [comment]
I thought I had added a waiver for this. I remember it installing and launching normally.

**Comment by rakleed** [comment]
> Commenter does not have sufficient privileges for PR 141806

@stephengillie

**Comment by stephengillie** [comment]
Completing validation.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**